### ISSUE TITLE
[0.75] Fabric e2etests for large snapshots are broken

### ISSUE THREAD

**Issue by TatianaKapos** [microsoft/react-native-windows#13436]
### Problem Description

Broken Tests
FlatList - Basic
ScrollView - refresh Control
SectionList - Scrollable

While working on the 0.75 branch these tests broke due to an Invalid string length error. I'm thinking it's because these are the only tests with 1000+ items and the toJson() function must have put a cap on what it can process but further investigation is needed. Check pages out locally and they render/work as intended

![image](https://github.com/microsoft/react-native-windows/assets/42554868/51e8e1c2-13c8-4c11-a988-adc2109ca3b6)

### Steps To Reproduce

1. Run tests on 0.75 branch
2. Look at Pipeline (https://dev.azure.com/ms/react-native-windows/_build/results?buildId=592440&view=logs&j=20253376-38f7-5084-b900-818d1e6181d3&t=e4bc6543-c5eb-5bc9-6033-5ab4b4216285)

### CLI version

npx react-native -v

### Environment

```markdown
npx react-native info
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

**Comment by Yajur-Grover** [comment]
This issue is also present on the 6/18 Integration, and I have commented the tests out pending a fix for this error. Verified the affected pages in the Tester App render as expected. However, on the SectionList example, when clicking on the 'scroll to' button to go to the listed section, all the buttons directed me to the top, rather than a specific individual section. @TatianaKapos did you see this on your preview branch as well?

**Comment by TatianaKapos** [comment]
@Yajur-Grover  let me double check! Are you seeing this on Paper/Fabric? We may need to see if it's coming from the integration or if it's been broken on main.

**Comment by Yajur-Grover** [comment]
Was seeing it on RNTesterApp-Fabric.

**Comment by jonthysell** [comment]
@TatianaKapos Has this been resloved for 0.75?

**Comment by TatianaKapos** [comment]
@jonthysell nope! We can put this on the backlog since this is also affecting main not just 0.75
